layer.md

```diff
@@ -1,63 +1,129 @@
-# Creating an Image Filesystem Changeset
+# Filesystem Layer Changeset
 
-An example of creating an Image Filesystem Changeset follows.
+A layer changeset is the distributable difference between filesystems.
+A layer changeset is not directly extracted, but applied to a filesystem.
```

Comment:
I think this is “don't extract this with generic tar tooling”.  I'd rather drop this line to avoid drawing a distinction between the undefined “directly extracted” and the undefined “applied” at this level.  You say pretty much the same thing [here][1], and at that point are actually defining application.

[1]: https://github.com/opencontainers/image-spec/pull/255/files#diff-dc2bb3b4eddcffe96e682193926e8ff6R156